common/src/main/java/org/bonitasoft/console/common/server/preferences/constants/WebBonitaConstantsUtils.java

```diff
@@ -162,27 +155,11 @@ private File getFolderFromBonitaHome(final String folderPath) {
     }
 
     private File getFolder(final String folderPath) {
-        return getFolder(new File(folderPath));
-    }
-
-    protected File getFolder(final File folder) {
+        final File folder = new File(folderPath);
         if (!folder.exists()) {
-            createFolderIfNecessary(folder);
-        }
-        return folder;
-    }
-
-    protected void createFolderIfNecessary(final File folder) {
-        if (webBonitaConstants instanceof WebBonitaConstantsImpl || tenantFolderExists()) {
             folder.mkdirs();
         }
-    }
-
-    protected boolean tenantFolderExists() {
```

Comment:
Carefull, in order to not create a folder for tenants that does not exists, we need to check the tenant is valid before creating the folder (security attack protection)